x-pack/plugins/security_solution/public/common/components/event_details/insights/insights.tsx

```diff
@@ -113,7 +115,8 @@ export const Insights = React.memo<Props>(
                 browserFields={browserFields}
                 data={sourceEventField}
                 eventId={eventId}
-                timelineId={timelineId}
+                scopeId={scopeId}
+                isInTimeline={isInTimeline}
```

Comment:
NIT: all these new `isInTimeline` props seem redundant, we are already passing the `scopeId` so we could check that inside those sub-components. We could also encapsulate this comparison in a helper `isInTimeline(scopeId: string) => boolean` function, in case someday we change the meaning of "being in timeline".

Reply:
also, we are using the same name for `const isInTimeline = isTimelineScope(scopeId)` in other places, which is not the same thing. Maybe we should call this one `isActiveTimeline`.